modules/material-management/test/multi_keyring.test.ts

```diff
@@ -358,6 +358,23 @@ describe('MultiKeyring: onDecrypt', () => {
     expect(unwrapDataKey(test.getUnencryptedDataKey())).to.deep.equal(unencryptedDataKey)
     expect(called).to.equal(true)
   })
+
```

Comment:
Same here, I think we want to test the case where `!material.hasValidKey() && !childKeyringErrors.length`